server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -584,6 +605,12 @@ private void setRetentionLeaseMillis(final TimeValue retentionLease) {
      */
     private volatile int maxRegexLength;
 
+    /**
+     * The max amount of time to wait for merges
+     */
+    private volatile TimeValue maxFullFlushMerge;
```

Comment:
Maybe include "WaitTime" in the variable name and methods so it is clear what this field is. Similar to how other fields in this class have "Size" and "Age" and "Interval" in their names.